github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -1211,6 +1211,9 @@ periodics:
       - /bin/sh
       - -c
       - |
+        # Build the perfscale-audit binary
+        make go-build
+
```

Comment:
why go-build instead of make bazel-build?

Reply:
Thought I changed this... it should be bazel-build